usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -674,7 +688,9 @@ def post(self, request, format=None):
                 account_number=F("federal_account_code"),
                 budgetary_resources=Subquery(
                     AppropriationAccountBalances.objects.filter(
-                        final_of_fy=True,
+                        submission__reporting_fiscal_year=fy,
+                        submission__submission_window__submission_reveal_date__lte=now(),
```

Comment:
I think this check might be redundant because the `is_final_balances_for_fy` field is only set to true after the submission is revealed. Taking this out could save a join.

Reply:
The submission window check was removed.